### ISSUE TITLE
XPath expression throws an exception when using mtom

### ISSUE THREAD

**Issue by lnu** [castlemock/castlemock#194]
I try to set an xpath expression to mock a specific response.
It does not work and I receive this in the logs:
```
ERROR: com.castlemock.core.basis.utility.XPathUtility - Unable to evaluate xpath expression
org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 1; Content is not allowed in prolog.
```
```
--uuid:f2de8049-8844-42aa-8d36-42c11327154c+id=2
Content-ID: <http://tempuri.org/0>
Content-Transfer-Encoding: 8bit
Content-Type: application/xop+xml;charset=utf-8;type="text/xml"

XML REMOVED
--uuid:f2de8049-8844-42aa-8d36-42c11327154c+id=2--
```
I can't really put the request here but I tried the xpath expression and it works.  
I'm running CastleMock in docker.

**Comment by karldahlgren** [comment]
Hi @lnu.
Would it be possible for you to create a request and XPath that might produce the same error? Perhaps remove all the sensitive data from the request? You can send me the request and XPath to my email directly if you prefer that: karl.dahlgren@castlemock.com

Thank you!
Best regards,
Karl Dahlgren

**Comment by lnu** [comment]
I made more tests and when I try the same mock from SoapUI it works. It seems whenever I use a wcf client (.net app or wcfstorm) it fails with the same error.

**Comment by karldahlgren** [comment]
Hi @lnu.
Thank you so much for reporting this issue. There was indeed an issue with MTOM and XPath. The bug has now been fixed and a new release has also been released (v1.45). The release is available on our webpage and on Docker.

Once again, thank you very much!
Best regards,
Karl Dahlgren
